javascript/stimulus_reflex.js

```diff
@@ -203,7 +189,9 @@ const register = (controller, options = {}) => {
             options['serializeForm'] = true
         }
 
-        const form = reflexElement.closest('form')
+        const form =
+          document.querySelector(reflexData.formSelector) ||
+          reflexElement.closest('form')
```

Comment:
```suggestion
        const form = reflexElement.closest(reflexData.formSelector || 'form')
```

I think we should still use `Element.closest()` to determine the form since there could be more than one form. Also since it's not mandatory to pass in an ID.

Reply:
hmm in my case this didn't work because it's not a child, but a sibling... reverting...

Reply:
But then I would recommend to do this instead:

```suggestion
        const form =
          reflexElement.closest(reflexData.formSelector) ||
          document.querySelector(reflexData.formSelector) ||
          reflexElement.closest('form')
```